### PR TITLE
settings framework in place

### PR DESCRIPTION
**Pull request by apandey16**

### CODE REVIEW

**Comment by apandey16** [comment]
Screenshot 2023-10-15 at 3 37 48 PM

File: Eat_and_Greet/src/screens/SettingsScreen/index.tsx [hunk 1]
```diff
@@ -1,20 +1,60 @@
...
-import styles from '../../style';
+  const toggleNotifications = () => {
+    setNotifications(!notifications);
+  };
+
+  const navigation = useNavigation();
+
+  return (
+    <View style={{ flex: 1, padding: 16, backgroundColor: '#7286D3' }}>
+      <View style={{ flex: 1, padding: 16, backgroundColor: '#E5E0FF' , borderRadius: 10}}>
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/screens/SettingsScreen/index.tsx:17]: Not necessary, but maybe throw these in a local style sheet.

File: Eat_and_Greet/src/screens/SettingsScreen/index.tsx [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+        
+        <List.Section>
+          <List.Subheader>General Settings</List.Subheader>
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/screens/SettingsScreen/index.tsx:20]: I'd maybe change the fontSize of the List.Subheaders to be a bit bigger, but that's a personal preference. Just feel they may be hard to see if we add a bunch of other settings items.

**Review by R303Vermeulen — APPROVED** [review]
Great work!

**Pull request merged**